src/components/Editor/EditorMenu.js

```diff
@@ -77,8 +76,7 @@ function getMenuItems(
     id: "node-menu-show-source",
     label: revealInTreeLabel,
     accesskey: revealInTreeKey,
-    disabled: false,
-    click: () => showSource(selectedSource.get("id"))
+    disabled: false
```

Comment:
haha - we want to keep the feature :)

Just remove the experimental feature flag